Plugins/Freemoji/Freemoji.plugin.js

```diff
@@ -56,13 +56,58 @@ module.exports = (() => {
                 note: 'Automatically splits messages containing emoji links so there won\'t be links in the middle of your messages.',
                 value: false
             },
+            {
+                type: 'slider',
+                id: 'splitDelay',
+                name: 'Split Message Delay',
+                note: 'The delay between each split message in ms.',
+                value: 100,
+                markers: [50, 100, 150, 200, 250, 500, 750, 1000],
+                stickToMarkers: true,
+                defaultValue: 100
+            },
+            {
+                type: 'switch',
+                id: 'splitDelaySlowdown',
+                name: 'Add Slowdown Time',
+                note: 'Adds the current slowmode time to the split message delay.',
+                value: true
+            },
+            {
+                type: 'slider',
+                id: 'splitLimit',
+                name: 'Split Message Limit',
+                note: 'Prevent the user from sending more than this many split messages at once. This setting and the one below is disabled if the value is set to zero.',
+                value: 4,
+                markers: [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10],
+                stickToMarkers: true,
+                defaultValue: 4
+            },
+            {
+                type: 'dropdown',
+                id: 'splitLimitMode',
+                name: 'Split Message Limit Mode',
+                note: 'What should happen to the rest of a message once the limit in the setting above is reached?',
+                value: true,
+                options: [
+                    {
+                        label: 'Send all the remaining text in one message',
+                        value: false
+                    },
+                    {
+                        label: 'Put the text back inside the message box',
+                        value: true
+                    }
+                ]
+            },
             {
                 type: 'slider',
                 id: 'emojiSize',
                 name: 'Emoji Size',
                 note: 'The size of the emoji in pixels. 48 is recommended because it is the size of regular Discord emoji.',
                 value: 48,
                 markers: [32, 40, 48, 60, 64, 80, 96],
+                defaultValue: 48,
```

Comment:
this isn't a thing

Reply:
Interesting, I guess it must have been caused by some other plugin then.
![default value](https://user-images.githubusercontent.com/39435218/183665802-3f030d02-a281-4a7a-9db9-e8e61a23e8d8.png)
As shown in this screenshot, the 'defaultValue' determines which value should be coloured green on my end.

Reply:
oh my bad then, good to know